encryption/frodo/frodokem-aes/frodokem-aes.yaml

```diff
@@ -1,3 +1,4 @@
 name: FrodoKEM-AES
+description: using AES for seed expansion. Superior iff hardware acceleration is available.
```

Comment:
Do we really need a mathematical description here? `iff` may be confusing for some people.

Reply:
I thought they might just think it's a typo ;)